app/extensions/safe/ffi/authenticator.js

```diff
@@ -512,7 +512,7 @@ class Authenticator extends SafeLib
                     ( userData, resultPtr, res ) =>
                     {
                         const result = resultPtr.deref();
-                        if ( result.error_code !== 0 && !res )
```

Comment:
`&& !res` removed because a `res` object is still sent with callback for `encode_auth_response` error.

Reply:
Just being curious, is it `res` an empty object in such a case?